addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -179,6 +180,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 if (command instanceof DecimalType) {
                     BigDecimal temperature = new BigDecimal(command.toString());
                     fritzBox.setSetTemp(ain, HeatingModel.fromCelsius(temperature));
+                } else if (command instanceof IncreaseDecreaseType) {
+                    BigDecimal settemp = (BigDecimal) getThing().getConfiguration().get(THING_SETTEMP);
+                    fritzBox.setSetTemp(ain, HeatingModel.fromCelsius(command.equals(IncreaseDecreaseType.INCREASE)
```

Comment:
I do not yet understand what is going on here it looks like add/subtract a fixed value from a fixed configuration value. So two times increase would have the same effect as once? Or am I missing something crucial here, which is likely I'm a little bit high on the anestecia of the dentist 😉